packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -44,7 +44,7 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
   })
 
   Commands.addAll({
-    switchToDomain<T> (originOrDomain: string, dataOrFn: T[] | (() => {}), fn?: (data?: T[]) => {}) {
+    switchToDomain<T> (originOrDomain: string, dataOrFn: { args: T } | (() => {}), fn?: (data?: T) => {}) {
```

Comment:
```suggestion
    switchToDomain<T> (originOrDomain: string, optionsOrFn: { args: T } | (() => {}), fn?: (args?: T) => {}) {
```

Reply:
updated in b06f21635af698d8d7a006e01ac1d04a699a4b42